### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by pfrollo**
I updated README as I learned about the process of schema sync and migration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,12 +134,12 @@ npm test
 ./run-tests.sh
 ````
 
-#### Options
+### Options
```
Comment (richard3d) [inline README.md:137]: Do these apply to local-dev also? May want to add them if not.

**Review by richard3d — APPROVED** [review]
Had one small comment, but looks good to me otherwise.

File: README.md (hunk 1 above)
Comment (pfrollo) [reply README.md:137]: Originally it was a subsection of Testing and I will make it such.

Comment (pfrollo) [reply README.md:137]: Never mind, it is a subsection....hmm...good question....

Comment (pfrollo) [reply README.md:137]: I don't know, I am leaving it as is.

**Pull request merged**